lib/nuget-parser/parsers/packages-config-parser.ts

```diff
@@ -27,14 +28,18 @@ export function parse(fileContent) {
   parseXML.parseString(fileContent, (err, result) => {
     if (err) {
       throw err;
-    } else {
-      const packages = result.packages.package || [];
-
-      packages.forEach(function scanPackagesConfigNode(node) {
-        const installedDependency = fromPackagesConfigEntry(node);
-        installedPackages.push(installedDependency);
-      });
     }
+    if (!('packages' in result)) {
```

Comment:
are you sure this syntax works? I never seen `in` used like this.

Reply:
nvm, I checked mdn

Reply:
I was more used to ` if (result.packages) `

Reply:
Actually not sure what is best, I come from Python where this is normal 🤣 Let's investigate.

Reply:
Seems to be even standard JS 🤷🏻 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#in

Reply:
```js
const trees = ["redwood", "bay", "cedar", "oak", "maple"];
"length" in trees; // returns true (length is an Array property)
```

😬